### ISSUE TITLE
Add "Frames" option to Add Timing Track

### ISSUE THREAD

**Issue by SopherConklin** [xLightsSequencer/xLights#4843]
Users may want to add a timing track that has a mark every frame.   This should be an option under Add Timing Track.
This would eliminate the requirement for the user to either correctly remember their framerate, or go to the sequence setting screen to look it up, neither of which they are guaranteed to do.

They may have tried to make all my sequences at 40 fps, but missed a few, which are still at 20 fps.   Now I think they are all at 40, so I would select 25ms from the list and get the wrong timing marks.   Or I may have a sequence at 24 or 30 fps because I am using a video effect with a video that is 24 or 30 fps and I want the video and xLights frames to align as close as possible.   If I open this sequence the next year, I might very well forget this, assume it's at 40 fps and select that for my frames time track.  

If the user wants a timeline with time marks at each frame we should make that as easy, convenient and foolproof as possible.  They select "Frames" from the list, the program knows what their actual fps is so it creates time marks at those intervals.  This might even allow a bit of simplification as possibly we could eliminate the now redundant "25ms" and "50ms" from the list in lieu of "Frames".   
This would make xLights more convenient, more intuitive, and less mistake prone.

**Comment by derwin12** [comment]
Zoom in .. they are already there.. 20 vs 40
![image](https://github.com/user-attachments/assets/df7d913f-0a24-4bb5-92bd-a937ce7f0fcf)

**Comment by SopherConklin** [comment]
The point wasn't that it can't already be done (for 20 and 40 fps and for others with Metronome)

The point is that first, there are other FPS in use and second,  we could put in one item which would cover all possibilities and eliminate a potential source of error as well.

**Comment by computergeek1507** [comment]
I don't see why someone would be added timing marks at 20fps or 40fps. Timing marks are normally used for the placement of effects and i don't think any user is placing effects every frame. The second use is for state or VU effect "cycling". If the user is cycling states that fast I think it's better to use a video or gif.

**Comment by SopherConklin** [comment]
You guys did add an option to add a Timing Track at 25ms or 50 ms, so the developers obviously found a use for that.   A usecase when I used timing marks at that resolution was my very first show, for Valentines Day.  I had assembled a composite video out of a diverse set of stuff that had very different color palettes.   I needed to know the exact frame that the video transitioned to the new piece, so that I could change the light colors to match in sync.   I believe I went into the Zoom Room and someone showed me how to insert Timing Every frame (I believe we used a Metronome because the 25ms/50ms wasn't there yet) and then go forward and back a frame at a time to find the transitions.   Worked very well.     Sometimes, when there is a simple fix to increase or simplify capability or to make errors less likely, you just put it in and the users will maybe find uses for it that the developers don't antipipate.
